Dockerfile

```diff
@@ -20,16 +20,23 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 		python-requests \
 	&& rm -rf /var/lib/apt/lists/* \
 	&& ln -s /usr/share/java/mysql-connector-java.jar "$CATALINA_HOME"/lib/ \
-	&& rm -rf $CATALINA_HOME/webapps/*m* 
-	
+	&& rm -rf $CATALINA_HOME/webapps/*m*
+
+ARG BRANCH=master
```

Comment:
Changing the default from an immutable tag that we tested to a mutable branch name will cause confusion, as it will create an image based on what the branch looked like the first time the image was built on this machine since the tomcat base image was updated.

In addition, the documented build process changes as new cBioPortal releases come out, so in order to keep the process to set up a basic installation relatively smooth for new users and our deployers, it's critical that this Dockerfile builds a predictable state of the cBioPortal code by default.